solidity/contracts/interfaces/IInterchainQueryRouter.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import "../middleware/libs/Call.sol";
+
+interface IInterchainQueryRouter{
```

Comment:
if we are going to have this explicit interface, would be nice to enforce this interface on the `InterchainQueryRouter` contract

Reply:
Have pushed required changes